mesonbuild/backend/vs2010backend.py

```diff
@@ -104,6 +103,7 @@ def generate_custom_generator_commands(self, target, parent_node):
                 exe = generator.get_exe()
                 infilelist = genlist.get_inputs()
                 outfilelist = genlist.get_outputs()
+                source_dir = os.path.join(self.environment.get_source_dir(), genlist.subdir)
```

Comment:
__[Flake8]__

local variable 'source_dir' is assigned to but never used


<!-- Comment by [SideCI](https://sideci.com) -->